genesis/process/shardGenesisBlockCreator.go

```diff
@@ -335,22 +335,24 @@ func createProcessorsForShard(arg ArgsGenesisBlockCreator) (*genesisProcessors,
 		return nil, err
 	}
 
-	transactionProcessor, err := transaction.NewTxProcessor(
-		arg.Accounts,
-		arg.Hasher,
-		arg.PubkeyConv,
-		arg.Marshalizer,
-		arg.SignMarshalizer,
-		arg.ShardCoordinator,
-		scProcessor,
-		genesisFeeHandler,
-		txTypeHandler,
-		genesisFeeHandler,
-		receiptTxInterim,
-		badTxInterim,
-		smartContract.NewArgumentParser(),
-		scForwarder,
-	)
+	argsNewTxProcessor := transaction.ArgsNewTxProcessor{
+		Accounts:          arg.Accounts,
+		Hasher:            arg.Hasher,
+		PubkeyConv:        arg.PubkeyConv,
+		Marshalizer:       arg.Marshalizer,
+		SignMarshalizer:   arg.SignMarshalizer,
+		ShardCoordinator:  arg.ShardCoordinator,
+		ScProcessor:       scProcessor,
+		TxFeeHandler:      genesisFeeHandler,
+		TxTypeHandler:     txTypeHandler,
+		EconomicsFee:      genesisFeeHandler,
+		ReceiptForwarder:  receiptTxInterim,
+		BadTxForwarder:    badTxInterim,
+		ArgsParser:        smartContract.NewArgumentParser(),
+		ScrForwarder:      scForwarder,
+		DisabledRelayedTx: false,
```

Comment:
Should this always be enabled on the genesisProcessor? No need for the config parameter?

Reply:
no need in genesis - there are no relayed txs in genesis.

Reply:
ok